allennlp/common/from_params.py

```diff
@@ -116,6 +116,37 @@ def create_kwargs(cls: Type[T], params: Params, **extras) -> Dict[str, Any]:
     params.assert_empty(cls.__name__)
     return kwargs
 
+
+def create_extras(cls: Type[T],
+                  extras: Dict[str, Any]) -> Dict[str, Any]:
+    """
+    Given a dictionary of extra arguments, returns a dictionary of
+    kwargs that actually are a part of the signature of the cls.from_params
+    (or cls) method.
+    """
+    subextras: Dict[str, Any] = {}
+    if hasattr(cls, "from_params"):
+        from_params_method = cls.from_params
```

Comment:
mypy is complaining about this line; you can just add `  # type: ignore` at the end, because mypy is wrong.